### PR TITLE
GROUP CAPSTONE PROJECT

### PR DESCRIPTION
**Pull request by prolajumokeoni**
In this capstone project we:
- Set up web pack.
- Created a movie website based on an external API with two interfaces: the home page and the comments popup.
- Counters in all the interfaces.
- Set up a JavaScript testing library (Jest).
- Functions that allow users to like and comment on movies.
- Used the Involvement API to log likes and comments.
-  Recorded a video describing the project and posted it on Youtube [here](https://youtu.be/J-EVUUZgiYA)

### https://mmatongo.github.io/group-capstone/dist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+# Group-Capstone
+
+> This is an API based webapp that displays movies and their titles. This is a single-page application(SPA) which contains the home page and comments modal pop-ups. Users can click the `thumbs up` icon to give a like to their favorite and post comments by clicking on the `comment` button in the modal.  
+
+# Demo Link
+
+Click [here](https://mmatongo.github.io/group-capstone/dist/) to view the website online.
+
+# Video Link 
+Click [here](https://youtu.be/X3809LCFK9s)
```
Comment (AshakaE) [inline README.md:12]: All the members in the video should appear in the video and **talk** about the project.

- [ ] Kindly record a new video where your all members of speak about the project 😄

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @prolajumokeoni  & @mmatongo  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Git flow ✔️ 
- Pull request ✔️ 
- REAMDE ✔️ 
- ES6 JavaScript ✔️ 
- Video presentation ❌ 


#  Required Changes ♻️ 🚧
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZawadiSumba66 — APPROVED** [review]
Hi Team 😃 , 
### STATUS: APPROVED 👍 
Amazing job guys :) Keep rocking 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mmatongo** [comment]
> Hi Team 😃 ,
> 
> ### STATUS: APPROVED 👍
> Amazing job guys :) Keep rocking 🚀
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> Congratulations! 🎉


Thank you.

**Pull request merged**